### DISCUSSION TITLE
Add support for multiple mysql instances

### DISCUSSION THREAD

**Discussion by nicholasrobertm** [pelican-dev/panel#14] [Ideas]
Currently with ptero when you setup a mysql instance for servers to use (in the databases tab) all servers no matter the node use the same instance. I'd be preferable to have the option to install mysql on each node and be able to select which physical database you want when creating databases in the databases tab. This could open the door for simple setups allowing them to have the servers on a node use a DB on the same node. 

Additionally it'd open the door for potentially using external database for some cases (e.g. RDS) which could be widely useful

**Comment by lancepioch** [comment]
Thanks, this would be a great idea for a plugin!
